spk/duplicity/Makefile

```diff
@@ -0,0 +1,34 @@
+SPK_NAME = duplicity
+SPK_VERS = 0.7.03
```

Comment:
Version 0.7.05 is out: http://duplicity.nongnu.org/index.html
Just to keep the package "bleeding edge" :)